std/process.d

```diff
@@ -1316,6 +1428,12 @@ enum Config
     process.
     */
     inheritFDs = 32,
+
+    /**
+    Spawn process in detached state. This removes the need in calling
+    $(LREF wait) to clean up the process resources.
```

Comment:
Should have a note here that calling `wait` or `kill` with the resulting `Pid` is invalid.